torchvision/datasets/caltech.py

```diff
@@ -30,6 +30,10 @@ class Caltech101(VisionDataset):
         download (bool, optional): If true, downloads the dataset from the internet and
             puts it in root directory. If dataset is already downloaded, it is not
             downloaded again.
+
+            .. warning::
+
+                To download the dataset `gdown <https://github.com/wkentaro/gdown>`_ is required.
```

Comment:
Just a prototype. When we agree on formatting etc. I'll add this to all datasets. I dislike a warning here somewhat, because that feels to strong. But that is what we currently do for other optional dependencies as well.

Reply:
https://docs-preview.pytorch.org/pytorch/vision/8237/generated/torchvision.datasets.Caltech101.html

![image](https://github.com/pytorch/vision/assets/6849766/7dab5d60-3a56-4572-b97f-54102c9a036e)

Reply:
Warning SGTM but if you prefer a `.. note` of something else, go ahead. Consistency isn't super critical here.